Makefile

```diff
@@ -98,24 +98,6 @@ ifeq ($(ZONE),0)
 	REGION_SUBS =ws://127.0.0.1:$(ZONE_$(REGION)_$(ZONE)_PORT_WS),,
 endif
 
-# Build specific prime, region, and zone commands for run-slice
```

Comment:
why did we delete all this ?

Reply:
None of these are used, they were only used for run-slice. You can search these variables they're never used in the makefile